examples/stat_object.py

```diff
@@ -36,7 +33,7 @@
 
 # Get object information of version-ID.
 result = client.stat_object(
-    "my-bucketname", "my-objectname",
+    "my-bucket", "my-object",
```

Comment:
They are all in comments, but there are still 33 references to `my-bucketname` and `my-objectname` in 21 files.
If you'd like, you may want to change them all to `my-bucket` and to `my-object` for consistency purposes.

Reply:
I don't find files containing those. Can you list those files?

Reply:
![Selection_025](https://user-images.githubusercontent.com/24552332/101062044-00a92500-3546-11eb-8e00-956433d6c7a2.jpg)

Reply:
None of them except `api.py` are in the repository. You would need to tidy up your work directory.